src/main/java/com/autotune/common/data/metrics/MetricResults.java

```diff
@@ -0,0 +1,79 @@
+package com.autotune.common.data.metrics;
```

Comment:
We need the LICENSE header to be added for the new file created.

Reply:
Added